polaris-core/src/main/java/org/apache/polaris/core/persistence/pagination/ReadFromStartPageToken.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.core.persistence.pagination;
+
+import java.util.List;
+
+/**
+ * This is needed until real pagination is implemented just to capture limit / page-size without any
+ * particular start position TODO: Remove this type
```

Comment:
This entire type can get removed once the `pageTokenBuilder` implementations are updated to return a more useful `PageToken` implementation. 

It's needed for now because we are updating interfaces to remove `int limit` in lieu of a `PageToken`, but `ReadEverythingPageToken` (which will stay for testing and for calls that want no pagination) doesn't support a limit.